### PR TITLE
1 Set Up the Question Model

### PR DESCRIPTION
**Pull request by leonardk1**
#1 
As A developer
I Want to set up the Question Model
So as to develop further features of the app

##### Tasks
- [x] Set up the minitest environment
- [x] Create and Test The Question model

### CODE REVIEW

**Comment by leonardk1** [comment]
@godfreym
@arafatm please review

**Comment by arafatm** [comment]
@leonardk1 questions don't make sense without answers. provide options for the answers

File: app/models/question.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Question < ActiveRecord::Base
+	validates :title, presence: true
```
Comment (godfreym) [inline app/models/question.rb]: @leonardk1 indentation

File: db/migrate/20160104161719_add_attributes_to_the_question_model.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddAttributesToTheQuestionModel < ActiveRecord::Migration
+  def change
+    add_column :questions, :title, :string
```
Comment (godfreym) [inline db/migrate/20160104161719_add_attributes_to_the_question_model.rb]: @leonardk1 the model is called Question and then there is an attribute called `questions`?

Comment (leonardk1) [inline db/migrate/20160104161719_add_attributes_to_the_question_model.rb]: @godfreym the attribute is called `:title` with a datatype of string

Comment (godfreym) [inline db/migrate/20160104161719_add_attributes_to_the_question_model.rb]: @leonardk1 Yeah sorry, I need :eyeglasses:

**Comment by leonardk1** [comment]
@arafatm the options have been added
@godfreym please review the changes

File: app/models/option.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class Option < ActiveRecord::Base
```
Comment (godfreym) [inline app/models/option.rb]: @leonardk1 what does option mean to Question ?

Comment (leonardk1) [inline app/models/option.rb]: @godfreym 
take a look at this example
```
Which FootBall Team do you support?
   a) Manchester United
   b) Leeds
   c) Aston Villa
```

the question is `which football team do you support?

the options are the answers

Comment (godfreym) [inline app/models/option.rb]: @leonardk1 OK cool, I didn't know if you know about Manchester United :smile: [kidding]

**Comment by godfreym** [comment]
@leonardk1 :+1:

**Comment by godfreym** [comment]
@arafatm Please Review, The progress from here will be easier if this is merged

**Comment by arafatm** [comment]
:+1:

**Pull request merged**